Feat/tp add trading example

timpara opened the pull request.

uniperJim committed:
changes to make it work with Azure hosted openAI models

timpara committed:
Add optimal execution trading example

Created a new example that demonstrates optimal trading execution:
- Implemented market simulator with price impact model
- Added evaluator with metrics for cost improvement vs baseline
- Fixed parameter ordering in simulate_execution function
- Included timeout handling for search algorithms

timpara committed:
Add optimal execution trading example

Created a new example that demonstrates optimal trading execution:
- Implemented market simulator with price impact model
- Added evaluator with metrics for cost improvement vs baseline
- Fixed parameter ordering in simulate_execution function
- Included timeout handling for search algorithms

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/codelion/openevolve?pullRequest=99) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/codelion/openevolve?pullRequest=99) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: uniperJim<br/>:x: timpara<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/codelion/openevolve?pullRequest=99) it.</sub>

timpara committed:
Update README.md

timpara committed:
Merge branch 'codelion:main' into feat/tp_add_trading_example

timpara committed:
refactor: rename optimia_execution to evolution_orchestrator

Rename folder to better reflect its role in coordinating the evolutionary
optimization process within the OpenEvolve system architecture.

timpara committed:
fix: adapt OpenAI client for AI Foundry compatibility

Modify API parameter handling in OpenAILLM to properly support AI Foundry endpoints:
- Add special case for AI Foundry endpoints
- Remove unsupported parameters for o-series models
- Update URL detection for Azure endpoints
- Improve parameter cleaning for different API variants

timpara committed:
fix: adapt OpenAI client for AI Foundry compatibility

Modify API parameter handling in OpenAILLM to properly support AI Foundry endpoints:
- Add special case for AI Foundry endpoints
- Remove unsupported parameters for o-series models
- Update URL detection for Azure endpoints
- Improve parameter cleaning for different API variants

timpara committed:
fix: adapt OpenAI client for AI Foundry compatibility

Modify API parameter handling in OpenAILLM to properly support AI Foundry endpoints:
- Add special case for AI Foundry endpoints
- Remove unsupported parameters for o-series models
- Update URL detection for Azure endpoints
- Improve parameter cleaning for different API variants

timpara committed:
fix: adapt OpenAI client for AI Foundry compatibility

Modify API parameter handling in OpenAILLM to properly support AI Foundry endpoints:
- Add special case for AI Foundry endpoints
- Remove unsupported parameters for o-series models
- Update URL detection for Azure endpoints
- Improve parameter cleaning for different API variants

codelion commented:
> License does not show up under license/cla

You will need to sign it here - https://cla-assistant.io/codelion/openevolve?pullRequest=99

adamlansky commented:
@timpara I would be very interested in having your PR merged, as it adds a very relevant example - so gently upping the license request from @codelion here
